client/layout/guided-tours/tours/jetpack-plugin-updates-tour/index.tsx

```diff
@@ -34,9 +35,25 @@ export const JetpackPluginUpdatesTour = makeTour(
 	>
 		<Step
 			name="init"
-			target=".plugin-item-jetpack .form-toggle__switch"
+			target={ JETPACK_TOGGLE_SELECTOR }
 			arrow="top-left"
 			placement="below"
+			wait={ () =>
+				new Promise( resolve => {
+					if ( document.querySelector( JETPACK_TOGGLE_SELECTOR ) ) {
+						return resolve();
+					}
```

Comment:
at this point since we're checking against the same query selector we might as well remove this conditional

```js
wait={ () => new Promise( resolve => {
	const waitForElement = () =>
		document.querySelector( JETPACK_TOGGLE_SELECTOR )
			? resolve()
			: setTimeout( waitForElement, 125 );

	waitForElement();
} ) }
```

or just to throw out there, use the good 'ole Y-combinator…(but don't really 😉 )

```js
wait={ () => new Promise( resolve => Y( f => () =>
	document.querySelector( JETPACK_TOGGLE_SELECTOR ) ? resolve() : setTimeout( f, 125 );
)() ) }
```